src/components/cart/CartToggle/CartToggle.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react'
 import Button from 'src/components/ui/Button'
-import { useCart } from 'src/sdk/cart/useCart'
 import { useCartToggleButton } from 'src/sdk/cart/useCartToggleButton'
+import { ShoppingCart } from 'phosphor-react'
+
+import './cart-toggle.scss'
 
 function CartToggle() {
   const btnProps = useCartToggleButton()
-  const { totalUniqueItems } = useCart()
 
   return (
-    <Button {...btnProps} className="mx-3">
-      cart: {totalUniqueItems}
+    <Button
+      {...btnProps}
+      className="cart-toggle"
+      aria-label="Button cart toggle"
```

Comment:
Do you think we can use another approach to show the quantity? Instead of using a `pseudo-element`, we could use `<span>`  for that. Or maybe you can just pass the quantity on the `aria-label`. Ex: `aria-label="Cart with 2 items"`

Reply:
I will pass in aria-label!